calls.go

```diff
@@ -0,0 +1,206 @@
+	values := url.Values{
+		"token":              {api.token},
+		"join_url":           {params.JoinURL},
+		"external_unique_id": {params.ExternalUniqueID},
+	}
+	if params.CreatedBy != "" {
+		values.Set("created_by", params.CreatedBy)
+	}
+	if params.DateStart != 0 {
+		values.Set("date_start", strconv.FormatInt(int64(params.DateStart), 10))
+	}
+	if params.DesktopAppJoinURL != "" {
+		values.Set("desktop_app_join_url", params.DesktopAppJoinURL)
+	}
+	if params.ExternalDisplayID != "" {
+		values.Set("external_display_id", params.ExternalDisplayID)
+	}
+	if params.Title != "" {
+		values.Set("title", params.Title)
+	}
+	if len(params.Users) > 0 {
+		data, err := json.Marshal(params.Users)
+		if err != nil {
+			return Call{}, err
+		}
+		values.Set("users", string(data))
+	}
+
+	response := &callResponse{}
+	if err := api.postMethod(ctx, "calls.add", values, response); err != nil {
+		return Call{}, err
+	}
+
+	return response.Call, response.Err()
+}
+
+// GetCallInfo returns information about a Call.
+func (api *Client) GetCall(callID string) (Call, error) {
+	return api.GetCallContext(context.Background(), callID)
+}
+
+// GetCallInfoContext returns information about a Call.
+func (api *Client) GetCallContext(ctx context.Context, callID string) (Call, error) {
+	values := url.Values{
+		"token": {api.token},
+		"id":    {callID},
+	}
+
+	response := &callResponse{}
+	if err := api.postMethod(ctx, "calls.info", values, response); err != nil {
+		return Call{}, err
+	}
+	return response.Call, response.Err()
+}
+
+func (api *Client) UpdateCall(callID string, params UpdateCallParameters) (Call, error) {
+	return api.UpdateCallContext(context.Background(), callID, params)
+}
+
+// UpdateCallContext updates a Call with the given parameters.
+func (api *Client) UpdateCallContext(ctx context.Context, callID string, params UpdateCallParameters) (Call, error) {
+	values := url.Values{
+		"token": {api.token},
+		"id":    {callID},
+	}
+
+	if params.DesktopAppJoinURL != "" {
+		values.Set("desktop_app_join_url", params.DesktopAppJoinURL)
+	}
+	if params.JoinURL != "" {
+		values.Set("join_url", params.JoinURL)
+	}
+	if params.Title != "" {
+		values.Set("title", params.Title)
+	}
+
+	response := &callResponse{}
+	if err := api.postMethod(ctx, "calls.update", values, response); err != nil {
+		return Call{}, err
+	}
+	return response.Call, response.Err()
+}
+
+// EndCall ends a Call.
+func (api *Client) EndCall(callID string) error {
+	return api.EndCallContext(context.Background(), callID)
+}
+
+// EndCallContext ends a Call.
+func (api *Client) EndCallContext(ctx context.Context, callID string) error {
+	values := url.Values{
+		"token": {api.token},
+		"id":    {callID},
+	}
+
+	response := &SlackResponse{}
+	if err := api.postMethod(ctx, "calls.end", values, response); err != nil {
+		return err
+	}
+	return response.Err()
+}
+
+// CallAddUsers adds users to a Call.
```

Comment:
How would you feel about renaming these functions to use `Participant` instead of `User` to more closely align to Slack's official terminology and API methods?